### PR TITLE
Translate blog articles about Kubecon NA 2023 and OSS NA 2023 into german

### PR DESCRIPTION
**Pull request by thelooter**
Also fixes a typo in  the article for OSS NA 2023

### CODE REVIEW

File: website/content/en/events/2023-oss-na.md [hunk 1]
```diff
@@ -4,13 +4,13 @@ description: TAG Environmental Sustainability presence at the Linux Foundation
...
 And we as a TAG have a few sessions planned on site!
 
 ### SustainabilityCon Sessions - [link](https://events.linuxfoundation.org/open-source-summit-north-america/program/schedule/)
 
 Are you interested in the impact your cloud projects and cloud-related work has on the environment?
 Then we would love to meet you at our [**TAG Environmental Sustainability Project Meeting**](https://tockify.com/cncf.public.events/detail/598/1683747900000) at OSS NA.
 
-We are looking forward to hearing your perspectives and welcome everyone who wants to contribute to our effort.
-Let’s reduce the environmental of the cloud, data centers and our projects.
+Wir freuen uns auf Ihre Ansichten und heißen jeden willkommen, der einen Beitrag zu unseren Bemühungen leisten möchte.
```
Comment (guidemetothemoon) [inline website/content/en/events/2023-oss-na.md:15]: This update of German content to the English version of the file is probably made by mistake?

**Review by guidemetothemoon — COMMENTED** [review]
Thanks for you contribution, @thelooter ! Added a minor request for change😊

File: website/content/en/events/2023-oss-na.md (hunk 1 above)
Comment (thelooter) [reply website/content/en/events/2023-oss-na.md:15]: Yes, this was a mistake, i probably copied it into the wrong file

Comment (guidemetothemoon) [reply website/content/en/events/2023-oss-na.md:15]: @thelooter can you revert this change back to English, please? Once it's done I can approve the PR :)

Comment (thelooter) [reply website/content/en/events/2023-oss-na.md]: Done, sorry it took so long

**Review by guidemetothemoon — APPROVED** [review]
Looks great, thank you @thelooter ! 💚

**Pull request merged**